### DISCUSSION TITLE
Concerns about telemetry shared info

### DISCUSSION THREAD

**Discussion by luisxkimo** [qetza/replacetokens-task#50] [Q&A] [answered]
Hi,

I would like to ask about the kind of information that is shared with the check "Send anonymous usage telemetry". The team have some security concerns about this before use this task on ADO.

Could be possible be more concise or clear about which kind of sensible information can be shared with this option?

Thanks

**Comment by qetza** [comment, marked as answer]
Hi @luisxkimo,
Al the data shared when you activate telemetry is explained here: https://github.com/qetza/replacetokens-task?tab=readme-ov-file#datatelemetry

To summarize:
- the parameters values are send exempt those which contains file paths)
- some numbers are send (number of source files, number of replacement done...)
- some identifiers are send (account id, project id and build definition id) anonymized as hash

**Reply by luisxkimo** [reply]
Hi @qetza,

Thanks for the clarification
